eleventy-navigation.js

```diff
@@ -74,22 +76,24 @@ function navigationToHtml(pages, options = {}) {
 
 	return pages.length ? `<${options.listElement}${!isChildList && options.listClass ? ` class="${options.listClass}"` : ''}>${pages.map(entry => {
 		let liClass = [];
+		let aClass = [ options.anchorClass ];
```

Comment:
I like this but can you test if `options.anchorClass` has a length first before appending it to `aClass`?